pkg/transform/transform.go

```diff
@@ -93,7 +84,6 @@ func Start() {
 
 	runner.Transform([]Transform{
 		APITransform{},
-		CrioTransform{},
```

Comment:
PR is straightforward, but this line. I hope it is intended change and the line was not removed accidentally.

Reply:
Nope, it was removed by accident. I just noticed it myself and corrected it. Thank you for catching it!